make/InterimImage.gmk

```diff
@@ -42,19 +43,17 @@ JLINK_TOOL := $(JLINK) -J-Djlink.debug=true \
     --module-path $(INTERIM_JMODS_DIR) \
     --endian $(OPENJDK_BUILD_CPU_ENDIAN)
 
-$(INTERIM_IMAGE_DIR)/$(JIMAGE_TARGET_FILE): $(JMODS) \
-    $(call DependOnVariable, INTERIM_MODULES_LIST)
-	$(call LogWarn, Creating interim jimage)
-	$(RM) -r $(INTERIM_IMAGE_DIR)
-	$(call MakeDir, $(INTERIM_IMAGE_DIR))
-	$(call ExecuteWithLog, $(INTERIM_IMAGE_DIR)/jlink, \
-	    $(JLINK_TOOL) \
-	    --output $(INTERIM_IMAGE_DIR) \
-	    --disable-plugin generate-jli-classes \
-	    --add-modules $(INTERIM_MODULES_LIST))
-	$(TOUCH) $@
-
-TARGETS += $(INTERIM_IMAGE_DIR)/$(JIMAGE_TARGET_FILE)
+$(eval $(call SetupExecute, jlink_interim_image, \
+    WARN := Creating interim jimage, \
+    DEPS :=  $(JMODS) $(call DependOnVariable, INTERIM_MODULES_LIST), \
+    OUTPUT_DIR := $(INTERIM_IMAGE_DIR), \
+    SUPPORT_DIR := $(INTERIM_JLINK_SUPPORT_DIR), \
+    COMMAND := $(RM) -r $(INTERIM_IMAGE_DIR) && $(JLINK_TOOL) --output $(INTERIM_IMAGE_DIR) \
```

Comment:
In Images.gmk, we use the `PRE_COMMAND` option for running `rm`. Maybe you want to use the same pattern here?

Reply:
That's a good suggestion. Thanks!